src/internal_modules/roc_address/target_berkley/roc_address/socket_addr.cpp

```diff
@@ -6,7 +6,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+#ifndef __WIN32__
 #include <arpa/inet.h>
+#endif
```

Comment:
Here and in other code, let's use `ROC_TARGET_POSIX` / `ROC_TARGET_WINDOWS` (whatever fits more natural) instead of `__WIN32__`.

E.g.:

```suggestion
#ifdef ROC_TARGET_POSIX
#include <arpa/inet.h>
#endif
```

Just to handle platform-specific code uniformly across the code base.

Reply:
Ok, starting with the easiest ones :) , done.